static/app/components/performance/searchBar.tsx

```diff
@@ -227,14 +229,39 @@ function SearchBar(props: SearchBarProps) {
           searchSubstring={searchString}
           loading={loading}
           items={searchResults}
-          onClick={handleSearch}
-          onIconClick={navigateToTransactionSummary}
+          onClick={handleChooseItem}
+          onIconClick={handleClickItemIcon}
         />
       )}
     </Container>
   );
 }
 
+const encodeItemToValue = (item: DataItem) => {
+  return `${item.transaction}:${item.project_id}`;
+};
+
+const decodeValueToItem = (value: string): DataItem => {
+  const lastIndex = value.lastIndexOf(':');
+
+  return {
+    project_id: parseInt(value.slice(lastIndex + 1), 10),
+    transaction: value.slice(0, lastIndex),
+  };
```

Comment:
Question, is lastIndexOf used here instead of split because the transaction can have a `:` in it?

Reply:
That was my assumption, though I didn't write the original code